docs/dev_guide/contents/dependencies.rst

```diff
@@ -0,0 +1,22 @@
+.. _dependency_versions:
+
+*************************
+Dependency Support Policy
+*************************
+
+.. note::
+
+    This policy is based on `NEP-0029`_.
+
+sunpy has a short list of core dependencies (Python, numpy, astropy, parfive) and a long list of optional dependencies.
+The minimum version of these packages that we enforce follows this policy.
+
+* Python: Released in the prior 42 months from the anticipated release date.
+* Astropy: Released in the prior 12 months from the anticipated release date.
+* Everything else: Released in the prior 24 months from the anticipated release date.
```

Comment:
I think we chatted briefly about this on Element, but I can't find it now. Why not make this 12 months too? If I remember correctly @Cadair has opinions.

Reply:
So that people don't have to upgrade the majority of our deps (e.g. numpy/mpl) in an otherwise nep 27 compliant env.